CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.7.2] - 2020-06-22
```

Comment:
you don't need to add the version, the bot will do it automatically after the PR is merged

Reply:
Ok, I misunderstood that. So, should I remove this whole line?

Reply:
I looked at the history of commits and I suppose so. I removed that line.